### PR TITLE
File browser:: use SweetAlert instead of alert method

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: app/scripts/modals/file-browser-modal-controller.js [hunk 1]
```diff
@@ -170,6 +178,7 @@ angular.module('workflow-variables').controller('FileBrowserModalCtrl', function
                        })
                        .error(function (xhr, status, error) {
                            SweetAlert.swal({
+                               title: "Deleted!",
```
Comment (amouhoub) [inline app/scripts/modals/file-browser-modal-controller.js:181]: I'm not sure the title here would be "Deleted!" as the text says Failed to delete the file ...
I suggest
Cannot delete file!
or
Oops!!!

File: app/scripts/modals/file-browser-modal-controller.js [hunk 2]
```diff
@@ -137,7 +141,11 @@ angular.module('workflow-variables').controller('FileBrowserModalCtrl', function
                     if(xhr) {
                         errorMessage = ": "+ xhr.errorMessage;
                     }
-                    alert("Failed to create the new folder " + pathname + errorMessage);
+                    SweetAlert.swal({
+                       title: "Create New Folder",
```
Comment (amouhoub) [inline app/scripts/modals/file-browser-modal-controller.js:145]: I believe the title of the message should not be Create New Folder as the text says  that it failed to create a new folder.
I suggest:
- Cannot create folder
or even better:
- Oops !!!

File: app/scripts/modals/file-browser-modal-controller.js [hunk 3]
```diff
@@ -107,7 +107,11 @@ angular.module('workflow-variables').controller('FileBrowserModalCtrl', function
                     if(xhr) {
                         errorMessage = ": "+ xhr.errorMessage;
                     }
-                    alert("Failed to upload the file " + selectedFile.name + errorMessage);
+                    SweetAlert.swal({
+                       title: "Upload",
```
Comment (amouhoub) [inline app/scripts/modals/file-browser-modal-controller.js:111]: Same as below,
Upload error
or Cannot upload file
or Oops!!!

File: app/scripts/modals/file-browser-modal-controller.js (hunk 2 above)
Comment (medou-boushab) [reply app/scripts/modals/file-browser-modal-controller.js:145]: yes, it'll be better
thanks

File: app/scripts/modals/file-browser-modal-controller.js (hunk 3 above)
Comment (medou-boushab) [reply app/scripts/modals/file-browser-modal-controller.js:111]: same

**Comment by amouhoub** [comment]
@medou-boushab You have my approval.
Please do a squash & merge,,, otherwise, it will pollute the tree with 27 commits !!!!

**Pull request merged**